integration/src/integration/java/pl/allegro/tech/hermes/integration/BasicAuthSubscribingTest.java

```diff
@@ -22,7 +26,7 @@ public void initializeAlways() {
     public void shouldAuthorizeUsingBasicAuthWhenSubscriptionHasCredentials() {
         // given
         Topic topic = operations.buildTopic("basicAuthGroup", "topic");
-        operations.createSubscription(topic, "subscription", "http://user:password@localhost:" + HTTP_ENDPOINT_PORT + "/" );
+        operations.createSubscription(topic, "subscription", "http://user:password@localhost:" + HTTP_ENDPOINT_PORT + "/");
```

Comment:
\[PMD\] INFO: The String literal "subscription" appears 4 times in this file; the first occurrence is on line 29